grudge/discretization.py

```diff
@@ -93,64 +98,20 @@ def __init__(self, array_context: ArrayContext, mesh: Mesh,
             to be carried out, or *None* to indicate that operations with this
             discretization tag should be carried out with the standard volume
             discretization.
+        :arg discr_from_dd: A mapping from discretization tags
+            from a :class:`grudge.dof_desc.DOFDesc`
+            to a :class:`meshmode.discretization.Discretization`.
+            At minimum, this should include a base discretization given by
+            the dof descriptor :class:`grudge.dof_desc.DD_VOLUME`.
+        :arg dist_boundary_connections: A dictionary whose keys denote the
+            partition group index and map to the appropriate face connections
+            for distributed boundaries, if any.
```

Comment:
How do the keys denote these things? Document their data type.

Reply:
Touched up the docs: https://github.com/inducer/grudge/pull/122/commits/6b8458c4cf71e73fa0352fb84cdaea5378bab418.